doc/src/ForManagers.dox

```diff
@@ -204,7 +207,7 @@
  * - Implement
  *   @ref openassetio.managerAPI.ManagerInterface.ManagerInterface.getRelatedReferences
  *   "getRelatedReferences" to return any appropriate @ref entity_reference
- *   "entity references" for the supplied @ref Specification. Hosts may
+ *   "entity references" for the supplied @ref trait_set. Hosts may
```

Comment:
[question] Is this correct? My understanding was that `getRelatedReferences` should still take a specification, so that its properties can be used as a  filter predicate. The signature in the code still refers to specs.

Reply:
Undone [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/249/commits/57505a503609a06d6fab8f6c7b8efd90ee3f48e6).